### ISSUE TITLE
"Previous page" key on the HTML index page goes next, not previous.

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#1765]
**Describe the bug**

The HTML report has shortcut keys. Two of them, are `[` and `]` for prev/next page.  On the index page, typing `]` will take you to the first source file page.  But `[` also takes you to the first source file page.  On a source file page, they behave correctly.

You can try it on the sample: https://nedbatchelder.com/files/sample_coverage_html/index.html

I can't understand why it's doing the wrong thing.

**Comment by jvalleroy** [comment]
In the sample page, lines 174 and 175, you have:
```
        <a id="prevFileLink" class="nav" href="d_7b071bdc2a35fa80_whiteutils_py.html"/>
        <a id="nextFileLink" class="nav" href="d_7b071bdc2a35fa80___init___py.html"/>
```

However, the "a" tag is not a void element, so you need to have a separate closing tag `</a>`.

**Comment by nedbat** [comment]
Amazing! TIL. Thanks so much.

**Comment by nedbat** [comment]
Fixed in 409938de9682b3502a185ad96d3676c66713b7c0.

**Comment by nedbat** [comment]
The commit on master is 74c87a821b8db7f0dc131e42756d58f3a7abc784.

**Comment by nedbat** [comment]
This is now released as part of [coverage 7.5.0](https://pypi.org/project/coverage/7.5.0).
